chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+
+#### 4. Transcription style
+Whether the text has punctuation, casing or both. If we want a system to generate fully formatted text that could be used for a publication or meeting transcription, we require training data with punctuation and casing. If we just require the spoken words in an unformatted structure, neither punctuation nor casing are necessary.
+
+A summary of datasets on the HF Hub:
+
+|                | Train Hours | Domain                                | Speaking Style         | Transcription Casing | Transcription Punctuation | Recommended Usage                                 |
+|----------------|-------------|---------------------------------------|------------------------|----------------------|---------------------------|---------------------------------------------------|
+| LibriSpeech    | 960         | Audiobooks                            | Narrated               | ❌                    | ❌                         | Academic benchmarks                               |
+| Common Voice 9 | 2224        | Wikipedia text + crowd-sourced speech | Narrated               | ✅                    | ✅                         | Day-to-day speech                                 |
+| TEDLIUM        | 452         | TED talks                             | Narrated               | ❌                    | ❌                         | Technical scientific, political and social topics |
+| Voxpopuli      | 543         | European Parliament recordings        | Spontaneous            | ❌                    | ❌                         | Non-native English speakers                       |
+| GigaSpeech     | 10000       | Audiobook, podcast, youtube           | Narrated & spontaneous | ❌                    | ✅                         | Robustness over multiple domains                  |
+| SPGISpeech     | 5000        | Financial meetings                    | Narrated & spontaneous | ✅                    | ✅                         | Fully formatted transcriptions                    |
+
+
+Let’s take an example where we want to train a speech recognition system to transcribe speech on the topic of machine learning (highly relevant!). Our speech is going to be narrated (scripted), and we’re not worried about punctuation or casing on our text output. From our reference table, it looks like TED-LIUM is a good choice of dataset!
+
+[//]: # On review, this sub-section might be too involved. Happy to omit it and dive straight into how to load a dataset.
+
+### How to load an audio dataset
+Before we can handle any audio datasets, we need to make sure we have the right dependencies installed. Specifically, we'll need the "Audio" feature from datasets: https://huggingface.co/docs/datasets/installation#audio. This should take care of the main Python packages we need to read audio files from byte form and convert them to arrays.
```

Comment:
We usually try to avoid having explicit URLs in the course and prefer having explicit code snippets for key steps like installation. In this case, you could have something like:

```suggestion
Before we can handle any audio datasets, we need to make sure we have the right dependencies installed. Specifically, we'll need the [`audio` feature](https://huggingface.co/docs/datasets/installation#audio) from 🤗 Datasets: 

```
pip install datasets[audio]
```

This should take care of the main Python packages we need to read audio files from byte form and convert them to arrays.
```

Reply:
Thanks for the heads up! Have followed your suggestion in https://github.com/huggingface/course/pull/294/commits/9d111104b597c83c0eb05f25f36544eed01eb3a0